pnr/nextpnr/fpga_interchange/build.sh

```diff
@@ -25,10 +25,10 @@ cmake -DARCH=fpga_interchange -DRAPIDWRIGHT_PATH=$RAPIDWRIGHT_PATH -DPYTHON_INTE
 make -j${CPU_COUNT}
 
 # List of devices available
+# TODO: the 200T device produces an out of memory error. Disabled for now
```

Comment:
Please include in the comment issue that I've just created to remember about it: https://github.com/hdl/conda-eda/issues/108

Reply:
Sure, done (also rebased and removed the WIP tag)